.gitignore

```diff
@@ -7,13 +7,18 @@ _build
 docs/_static/pypi.svg
 .tox
 __pycache__
+
+# Packaging artifacts
 black.egg-info
+black.dist-info
```

Comment:
I've been spending a lot time recently in the Python packaging land hence this addition :)